azure-pipelines.yml

```diff
@@ -813,7 +813,7 @@ jobs:
           tell_daml() {
               local message pr_handler
               message=$1
-              pr_handler=$(head -1 release/rotation | awk "${print $1}")
+              pr_handler=$(head -1 release/rotation | awk '${print $1}')
```

Comment:
Wait wait wait. Why is there a `$` there at all?

It should be `awk '{ print $1 }'`.

Reply:
good catch, https://github.com/digital-asset/daml/pull/7104